packages/react-reconciler/src/ReactProfilerTimer.js

```diff
@@ -404,6 +429,24 @@ export function recordEffectDuration(fiber: Fiber): void {
   }
 }
 
+export function recordEffectError(errorInfo: CapturedValue<mixed>): void {
+  if (!enableProfilerTimer || !enableProfilerCommitHooks) {
```

Comment:
Should this have a check for `enableComponentPerformanceTrack`? I don't know what all these flags do, but the calls above are flagged behind that and not `enableProfilerCommitHooks`

Reply:
It's a little unclear but if the *other* DevTools profiler wanted to use this information it could as long as `enableProfilerCommitHooks` was enabled. It's just not used today.

We should really probably just make sure that the `enableProfilerCommitHooks` flag is always gated by `enableProfilerTimer`. That way it can be always true. Since it's always true we can just remove and ship the `enableProfilerCommitHooks` flag to simplify things a bit.